sklearn/preprocessing/_data.py

```diff
@@ -3205,6 +3205,18 @@ def power_transform(X, method='yeo-johnson', *, standardize=True, copy=True):
 
     Read more in the :ref:`User Guide <preprocessing_transformer>`.
 
+        .. warning:: Risk of data leak
```

Comment:
the block below should be indented, but not the header

It should look like this:

```
.. warning:: Risk of data leak
	indented here
```

Reply:
got it, thank you for your help